internal/xds/translator/httpfilters.go

```diff
@@ -91,28 +95,28 @@ func newOrderedHTTPFilter(filter *hcmv3.HttpFilter) *OrderedHTTPFilter {
 	// When the fault filter is configured to be at the first, the computation of
 	// the remaining filters is skipped when rejected early
 	switch {
-	case filter.Name == wellknown.Fault:
+	case isFilterType(filter, wellknown.Fault):
 		order = 1
-	case filter.Name == wellknown.CORS:
+	case isFilterType(filter, wellknown.CORS):
 		order = 2
 	case isFilterType(filter, extAuthFilter):
 		order = 3
-	case filter.Name == basicAuthFilter:
+	case isFilterType(filter, basicAuthFilter):
 		order = 4
 	case isFilterType(filter, oauth2Filter):
 		order = 5
-	case filter.Name == jwtAuthn:
+	case isFilterType(filter, jwtAuthn):
 		order = 6
 	case isFilterType(filter, extProcFilter):
-		order = 7
+		order = 100 + mustFilterIndex(filter.Name)
```

Comment:
curious around why the num changed here, did the logic change, are we trying to create a hole to accomodate more filters ?

Reply:
Because there're more than one wasm/extproc filters for an HTTPRoute.